cli-e2e-test/rel/device.rel

```diff
@@ -0,0 +1,10 @@
+bound import_config:device_data:schema
```

Comment:
```suggestion
bound import_config:device_data:schema = Symbol, String
```

Reply:
fixed

Reply:
Reverted since Symbol is not a type and this relation is needed just for stub.